src/main/java/AlexLink/HomeWork/AbstractClass/FlyTransport.java

```diff
@@ -0,0 +1,38 @@
+package AlexLink.HomeWork.AbstractClass;
+
+public abstract class FlyTransport extends Transport {
+
+    private int wingspan;
+    private int minLengthRunway;
+
+    public FlyTransport(int power, int maxSpeed, int weight, String model, int wingspan, int minLengthRunway) {
+        super(power, maxSpeed, weight, model);
+        if (wingspan > 10 && wingspan < 60) this.wingspan = wingspan;
```

Comment:
Use here existing methods **setWingspan** and **setMinLengthRunway**.
Because you duplicate logic.

Reply:
This is for ALL classes, where it repeat

Reply:
fixed